addons/hr_expense/tests/test_expenses.py

```diff
@@ -211,17 +211,14 @@ def test_expense_main_flow(self):
             (self.analytic_account_1 | self.analytic_account_2).unlink()
 
         # Unlinking moves
-        (payment_1 | payment_2).move_id.button_draft()
+        (payment_1 | payment_2).action_draft()
         self.assertRecordValues(expense_sheet_by_employee, [{'payment_state': 'not_paid', 'state': 'post'}])
         expense_sheet_by_employee.account_move_ids.button_draft()
         expense_sheet_by_employee.account_move_ids.unlink()
```

Comment:
Still necessary ?

Reply:
I'd add an assert to verify that we correctly unlinked the move

Reply:
> Still necessary ?

Yesss the moves from the expense_sheet are not the same as the one from the payments